jupyterhub_moss/spawner.py

```diff
@@ -1,17 +1,28 @@
+from __future__ import annotations
```

Comment:
[pep-0585](https://peps.python.org/pep-0585/#implementation): allows to use `dict` and `list`... instead of `typing.Dict` and `typing.List` with python<3.9